ntbs-integration-tests/NotificationPages/OverviewPageTests.cs

```diff
@@ -196,27 +222,46 @@ public async Task Get_ReturnsOverviewPageReadOnlyVersion_ForUserWithReadOnlyPerm
                 var document = await GetDocumentAsync(response);
 
                 // Assert
-                Assert.NotNull(document.QuerySelectorAll("#patient-details-overview-header"));
-                Assert.Null(document.QuerySelector("#patient-details-edit-link"));
+                Assert.NotNull(document.QuerySelector($"#{PatientDetailsOverviewSectionId}-title"));
+                Assert.Null(document.QuerySelector($"#{PatientDetailsOverviewSectionId}-edit-link"));
             }
         }
 
+        /* Just found that ".WithNotificationAndPostcodeConnected" has never worked.

```

Comment:
I am planning to look into and fix this, just thought it wasn't worth not getting the rest of it into review. Also my comment is probably not true, it may have worked in the past but it doesn't anymore.